build-system/git.js

```diff
@@ -52,12 +55,12 @@ exports.gitTravisMasterBaseline = function() {
 };
 
 /**
- * Shortens a commit SHA to 7 characters for human readability.
+ * Shortens a commit SHA to 9 characters (git default) for human readability.
```

Comment:
GitHub uses 7, which is why I chose that number in the first place

Reply:
Aha! It turns out that Github uses 7, and Travis logging uses 9. I've added a `length` param to the function. Visual tests will continue to use 7, while Travis logging will use 9 so it's easy to do searches for SHA values in the logs.

Reply:
Damn, it appears `git` uses different standards in different places. (My linux workstation uses 9.)

Never mind, I've reverted this change.